src/main/java/io/debezium/connector/vitess/Vgtid.java

```diff
@@ -5,23 +5,30 @@
  */
 package io.debezium.connector.vitess;
 
-import java.util.HashSet;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
-import java.util.Set;
 
-import com.google.gson.Gson;
-import com.google.gson.reflect.TypeToken;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 import binlogdata.Binlogdata;
 
-/** Vitess source position coordiates. */
+/** Vitess source position coordinates. */
 public class Vgtid {
     public static final String CURRENT_GTID = "current";
-    private static final Gson gson = new Gson();
+    public static final String KEYSPACE_KEY = "keyspace";
+    public static final String SHARD_KEY = "shard";
+    public static final String GTID_KEY = "gtid";
```

Comment:
```suggestion
    private static final String KEYSPACE_KEY = "keyspace";
    private static final String SHARD_KEY = "shard";
    private static final String GTID_KEY = "gtid";
```

Reply:
I made them public because if users want to deserialize Vgtid they can have access to the keys as well.